meta-dts-distro/recipes-dts/dts/dasharo-deploy/dasharo-deploy

```diff
@@ -241,10 +241,8 @@ bootsplash_migration() {
     # unnecessary messages
     cbfstool /tmp/dasharo_dump.rom extract -r BOOTSPLASH -n logo.bmp -f /tmp/logo.bmp >> $ERR_LOG_FILE 2>&1 || return 1
     echo -n "Backing up custom boot logo... "
-    cbfstool $BIOS_UPDATE_FILE remove -n logo.bmp -r BOOTSPLASH >> $ERR_LOG_FILE 2>&1 || \
-      print_warning "Failed! Default boot splash will be used."
-    cbfstool $BIOS_UPDATE_FILE add -f /tmp/logo.bmp -r BOOTSPLASH -n logo.bmp -t raw -c lzma >> $ERR_LOG_FILE 2>&1 || \
-      print_warning "Failed! Default boot splash will be used."
+    dcu logo /tmp/dasharo_dump.rom -l /tmp/logo.bmp >> $ERR_LOG_FILE 2>&1 || \
```

Comment:
It is dump, or the `$BIOS_UPDATE_FILE` going to be flashed?

Reply:
BIOS_UPDATE_FILE obviously... fixed: https://github.com/Dasharo/meta-dts/pull/84/commits/0f78c126c0c9244fa2f6301bdabe990f9ef676bb